lighthouse-cli/cli-flags.js

```diff
@@ -70,114 +64,218 @@ function getFlags(manualArgv) {
-          'only-audits', 'only-categories', 'skip-audits', 'budget-path',
-        ],
-        'Configuration:')
+      // Output
+      .options({
+        'output': {
+          type: 'array',
+          default: /** @type {['html']} */ (['html']),
+          coerce: coerceOutput,
+          describe: 'Reporter for the results, supports multiple values. choices: "json", "html", "csv"',
+        },
+        'output-path': {
+          type: 'string',
+          describe: `The file path to output the results. Use 'stdout' to write to stdout.
+  If using JSON output, default is stdout.
+  If using HTML or CSV output, default is a file in the working directory with a name based on the test URL and date.
+  If using multiple outputs, --output-path is appended with the standard extension for each output type. "reports/my-run" -> "reports/my-run.report.html", "reports/my-run.report.json", etc.
+  Example: --output-path=./lighthouse-results.html`,
+        },
+        'view': {
+          type: 'boolean',
+          default: false,
+          describe: 'Open HTML report in your browser',
+        },
+      })
+      .group(['output', 'output-path', 'view'], 'Output:')
+
+      // Other options.
+      .options({
+        'locale': {
+          coerce: coerceLocale,
+          describe: 'The locale/language the report should be formatted in',
+        },
+        'blocked-url-patterns': {
+          array: true,
+          type: 'string',
+          describe: 'Block any network requests to the specified URL patterns',
+        },
+        'disable-storage-reset': {
+          type: 'boolean',
+          describe: 'Disable clearing the browser cache and other storage APIs before a run',
+        },
+        'throttling-method': {
+          type: 'string',
+          describe: 'Controls throttling method',
+        },
+      })
       .describe({
-        'cli-flags-path': 'The path to a JSON file that contains the desired CLI flags to apply. Flags specified at the command line will still override the file-based ones.',
-        // We don't allowlist specific locales. Why? So we can support the user who requests 'es-MX' (unsupported) and we'll fall back to 'es' (supported)
-        'locale': 'The locale/language the report should be formatted in',
-        'enable-error-reporting':
-            'Enables error reporting, overriding any saved preference. --no-enable-error-reporting will do the opposite. More: https://git.io/vFFTO',
-        'blocked-url-patterns': 'Block any network requests to the specified URL patterns',
-        'disable-storage-reset':
-            'Disable clearing the browser cache and other storage APIs before a run',
-        'emulated-form-factor': 'Controls the emulated device form factor (mobile vs. desktop) if not disabled',
-        'throttling-method': 'Controls throttling method',
         'throttling.rttMs': 'Controls simulated network RTT (TCP layer)',
```

Comment:
> Why does this need to stay in a `describe` block?

because object support in yargs is [half baked](https://github.com/yargs/yargs/issues/1230) and I was putting off figuring out what to do with these and then I forgot to go back and do something with them :) I'll fix.